code/frameworks/nextjs-vite/package.json

```diff
@@ -108,7 +108,7 @@
     "@storybook/react": "workspace:*",
     "@storybook/react-vite": "workspace:*",
     "styled-jsx": "5.1.6",
-    "vite-plugin-storybook-nextjs": "^2.0.3"
+    "vite-plugin-storybook-nextjs": "2.0.5"
```

Comment:
```suggestion
    "vite-plugin-storybook-nextjs": "^2.0.5"
```

Reply:
@valentinpalkovic 
Thanks for reviewing!

Fixed by https://github.com/storybookjs/storybook/pull/31906/commits/195b8ff9888fc239dce6ac95aae6029b7a56bfec